src/modules/auth/passwd/ForgetPwdScreen.js

```diff
@@ -35,9 +35,11 @@ function ForgetPwdCard(props) {
 	const [ completed, setCompleted ] = useState(false);
 	let invalidCode = getParams("invalid_code");
 
-	const identRegister = register('ident');
+	const usernameRegister = register('username');
 
 	const onSubmit = async (values) => {
+		values.ident = values.username;
```

Comment:
@petrKavulok I would stick to the value assignment as you have on line `47` to have it unified within the method

```
values["ident"] = values["username"];
delete values["username"];
```

Btw. also write some comment pls why we are doing this "substitution"